### ISSUE TITLE
Finish this

### ISSUE THREAD

**Issue by goto-bus-stop** [browserify/perf-hooks-browserify#1]
opening an issue to remind me about it

**Comment by adamgen** [comment]
Yep, it was disappointing to see that this package is not on npm after following the readme

**Comment by chrisidakwo** [comment]
I believe @goto-bus-stop is probably dead!

Sad!

We could have had a life saver here!

RIP to a real one! 🍻
